File: UVA/Ch 5. Mathematics/10523 - Very Easy.py
# Problem: 10523 - Very Easy !!!

'''
Author: DNS404
'''

import sys
# import math


inputs = []
while True:
    try:
        line = input()
    except EOFError:
        break
    if line == "":
        break
    no1, no2 = line.split()
    line = [int(no1), int(no2)]
    inputs.append(line)

for i in range(0, len(inputs)):
    s = 0
    for j in range(1, inputs[i][0]+1):
        s += j * pow(inputs[i][1], j)
    print(s)

File: UVA/Ch 5. Mathematics/10523 - Very Easy.java
// Problem: 10523 - Very Easy !!!

import java.util.Scanner;
import java.math.BigInteger;

class Main {
  public static void main(String[] args){
    Scanner sc = new Scanner(System.in);
    while(sc.hasNextInt()){
      int n = sc.nextInt();
      int a = sc.nextInt();
      BigInteger ans = new BigInteger("0");
      for(int i=1; i<=n; i++){
        BigInteger temp = BigInteger.valueOf(a);
        temp = temp.pow(i);
        temp = temp.multiply(BigInteger.valueOf(i));
        ans = ans.add(temp);
      }
      System.out.println(ans);
    }
  }
}
